HelpAssistHer/help-assist-her
# Style team component

## Description
Add the team component with images and bios to the About page.

Note: the images and text are all just placeholders for now. 😆 

Also note: the images should scroll from left to right and when you click on an image, it will display the name, title, and bio for that person.

<img width="386" alt="Screen Shot 2019-04-10 at 10 32 32 PM" src="https://user-images.githubusercontent.com/6378435/55933000-9f01ad80-5be0-11e9-9260-f698253391a5.png">

## Test Plan
1. Go to http://localhost:4000/about
1. See the team component (shown above) and verify it matches designs
1. Make sure it looks good on both desktop and mobile views
